### DISCUSSION TITLE
How to set `plugins.blog: true`?

### DISCUSSION THREAD

**Discussion by rockbenben** [vuepress-theme-hope/vuepress-theme-hope#2711] [Q&A] [answered]
Notification in the latest deployment: "Blog feature is tree-shakable in v2, you should set plugins.blog: true in theme options to enable it."

As such, I made adjustments to the theme.ts settings, yet the same notification still presented itself. Could my configuration be in error? 

```
  plugins: {
    blog: true,
```

Upon perusal of [the blog sample page](https://github.com/Mister-Hope/Mister-Hope.github.io/blob/main/src/.vuepress/theme.ts), I noticed that there was no additional setting for 'plugins.blog: true'.

**Comment by Mister-Hope** [comment, marked as answer]
https://github.com/Mister-Hope/Mister-Hope.github.io/blob/8cad44d0e924d97beb0a2f99e222db12e01ba8e0/src/.vuepress/theme.ts#L79-L81

https://theme-hope.vuejs.press/guide/blog/intro.html

https://theme-hope.vuejs.press/config/plugins/blog.html

You may and you are expected to read docs carefully
